test/excludes/ActiveRecord/AdapterForeignKeyTest.rb

```diff
@@ -1,2 +1,4 @@
 exclude :test_foreign_key_violations_are_translated_to_specific_exception_with_validate_false, "Skipping until we can triage further. See https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/48"
 exclude :test_foreign_key_violations_are_translated_to_specific_exception, "Skipping until we can triage further. See https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/48"
```

Comment:
can you override these two tests to see if they pass?

Reply:
One of these tests are completely removed from the test suit, the other one I override